api/api.proto

```diff
@@ -1864,6 +1864,15 @@ service OpenStorageCloudBackup {
       };
     }
 
+  // Update existing cloud backup schedule
+  rpc SchedUpdate(SdkCloudBackupSchedUpdateRequest)
+    returns (SdkCloudBackupSchedUpdateResponse) {
+      option(google.api.http) = {
+        post: "/v1/cloudbackups/schedules"
```

Comment:
`post` is for create operations. `put` should be used for updates.

Reply:
I missed it. Thanks! Done